### ISSUE TITLE
Update devDependencies

### ISSUE THREAD

**Issue by nelsonic** [dwyl/aws-lambda-deploy#64]
The badges in this repo are looking a bit sad:
![image](https://user-images.githubusercontent.com/194400/73615343-fe60fc00-45fe-11ea-9cd4-d9dc9b8ad5dd.png)

## `devDependencies`

Quite a few `devDependencies` are out-of-date (_not surprising given last update was **`28 Sep 2017`** ⏳ 
https://david-dm.org/dwyl/aws-lambda-deploy?type=dev
![image](https://user-images.githubusercontent.com/194400/73615377-3405e500-45ff-11ea-8609-84880b2da618.png)

## `dependencies`

We can probably also update the `dependencies` while we're at it. 💭 
https://david-dm.org/dwyl/aws-lambda-deploy
![image](https://user-images.githubusercontent.com/194400/73615388-4a13a580-45ff-11ea-92e1-d6c5dd5961c2.png)

## Code Climate: ![Code Climate maintainability](https://img.shields.io/codeclimate/maintainability/dwyl/aws-lambda-deploy?color=brightgreen&style=flat-square)

```
![Code Climate maintainability](https://img.shields.io/codeclimate/maintainability/dwyl/aws-lambda-deploy?color=brightgreen&style=flat-square)
```
Relates to: #63 

I think it would be good to publish a **_maintenance_ release** of this package to confirm that everything is still working ahead of working on https://github.com/dwyl/aws-ses-lambda/pull/4 

# Todo

+ [x] update depenedncies
+ [x] remove `babel`
+ [x] run tests
+ [x] deploy a lambda!
+ [x] publish new ~~`minor`~~ **`major`** version.

**Comment by nelsonic** [comment]
Apparently `eslint` does not allow us to use `assert.equal` anymore ...
![image](https://user-images.githubusercontent.com/194400/74196177-993b8500-4c54-11ea-9f96-31e57fd275ef.png)

![image](https://user-images.githubusercontent.com/194400/74196222-b1ab9f80-4c54-11ea-9a9a-3b3dc6761745.png)

**Comment by nelsonic** [comment]
Same again for `assert.deepEqual` > `assert.deepStrictEqual` 🙄 
![image](https://user-images.githubusercontent.com/194400/74196441-241c7f80-4c55-11ea-9c6d-bf8dcb2f5890.png)

**Comment by nelsonic** [comment]
Latest version published to NPM `dpl@4.0.0` :shipit:
